ansible/roles/mn-evo-services/templates/dapi-envoy.yaml.j2

```diff
@@ -20,12 +20,12 @@ static_resources:
                   cluster: echo_service
```

Comment:
As I understood it's only for Core and Platfrom services which are listening on 3005 port. We also have TransactionsFilterStream on 3006, so we need to add one more listener?

Reply:
yes I think

Reply:
It seems like we need just to add one more virtual host and cluster for TransactionsFilterStream